ui/public/locales/pt_BR.json

```diff
@@ -1,879 +1,1143 @@
+"label.add.firewall": "Adicionar regra de firewall",
 "label.add.globo.dns": "Adicionar GloboDNS",
 "label.add.gslb": "Adicionar GSLB",
 "label.add.guest.network": "Adicionar rede guest",
-"label.add.host": "Adicionar Host",
-"label.add.ingress.rule": "Adicionar Regra de Entrada",
+"label.add.host": "Adicionar host",
+"label.add.ingress.rule": "Adicionar regra de entrada",
 "label.add.intermediate.certificate": "Adicionar certificado intermedi\u00e1rio",
-"label.add.internal.lb": "Adiciona LB Interno",
-"label.add.ip.range": "Adicionar Range de IP",
-"label.add.isolated.guest.network": "Adiciona Rede Guest Isolada",
-"label.add.isolated.guest.network.with.sourcenat": "Adicionar rede Guest isolada com SourceNat",
-"label.add.isolated.network": "Adiciona Rede Isolada",
-"label.add.l2.guest.network": "Add L2 Guest Network",
-"label.add.ldap.account": "Adicionar Conta LDAP",
-"label.add.list.name": "Nome de Lista ACL",
-"label.add.load.balancer": "Adicionar Load Balance",
-"label.add.more": "Adicionar Mais",
+"label.add.internal.lb": "Adicionar LB interno",
+"label.add.ip.range": "Adicionar intervalo de IPs",
+"label.add.isolated.guest.network": "Adicionar rede guest isolada",
+"label.add.isolated.guest.network.with.sourcenat": "Adicionar rede guest isolada com sourceNat",
+"label.add.isolated.network": "Adiciona rede isolada",
+"label.add.kubernetes.cluster": "Adicionar cluster Kubernetes",
+"label.add.l2.guest.network": "Adicionar rede guest L2",
+"label.add.ldap.account": "Adicionar conta LDAP",
+"label.add.ldap.list.users": "Listar usu\u00e1rios LDAP",
+"label.add.list.name": "Nome da lista ACL",
+"label.add.load.balancer": "Adicionar balanceador de carga",
+"label.add.management.ip.range": "Adicionar intervalo IP do management",
+"label.add.more": "Adicionar mais",
 "label.add.netscaler.device": "Adicionar dispositivo Netscaler",
-"label.add.network": "Adicionar Rede",
+"label.add.network": "Adicionar rede",
 "label.add.network.acl": "Adicione ACL de rede",
-"label.add.network.acl.list": "Adicionar Lista de ACL de Rede",
-"label.add.network.device": "Adicionar Dispositivo de Rede",
+"label.add.network.acl.list": "Adicionar lista de ACL de rede",
+"label.add.network.device": "Adicionar dispositivo de rede",
 "label.add.network.offering": "Adicionar oferta de rede",
 "label.add.new.f5": "Adicionar um novo F5",
 "label.add.new.gateway": "Adicionar novo gateway",
+"label.add.new.iso": "Adicionar nova ISO",
 "label.add.new.netscaler": "Adicionar um novo NetScaler",
 "label.add.new.pa": "Adicionar novo Palo Alto",
 "label.add.new.srx": "Adicionar um novo SRX",
 "label.add.new.tier": "Adicionar nova camada",
-"label.add.nfs.secondary.staging.store": "Adiciona Armazenamento NFS de Est\u00e1gio Secund\u00e1rio",
-"label.add.niciranvp.device": "Adicionar Controlador Nvp",
-"label.add.opendaylight.device": "Adiciona Controlador OpenDaylight",
+"label.add.nfs.secondary.staging.store": "Adicionar armazenamento NFS de est\u00e1gio secund\u00e1rio",
```

Comment:
I know that it was already translated to "Estágio Secundário".
But I don't think that `staging` would be translated into `Estágio`.
Normally Staging is for "testing", and can be literally translated to "encenação".

PS: I am curious where it is located in the UI. To be honest I have no idea.

Reply:
You're right, to be honest, the best fit here would be to let as `staging secundário`, what do you think?